### PR TITLE
Closes #40 create result accordion

### LINKED ISSUE #40: Create ResultAccordion component
Description:
This is the component that displays the results for a single topic. Use the UIKit Accordion component and pass it a Topic component and a Content component. Use Styled Components for Topic and Content (or Styled Component wrappers).

Acceptance Criteria:
 - [ ] Displays the Topic name
 - [ ] Is expanded by default
 - [ ] Displays expand/collapse toggle:
      - [ ] State 1: When the box is expanded:
                    -  an expand collapse toggle
                    -  The toggle is titled "Show Less"
                    -  On click, the box is collapsed (see state 2)
                    -  Results text displays for all "yes" answers for that topic
                    -  Where available, "what does the law say about this" link displays for each "yes" answer for that topic
                   - "What does the law say about this" link goes to the right place.
      - [ ] State 2: When the box is collapsed:
                   - an expand collapse toggle
                   * The toggle is titled "Show More"
                   * On click, the box is expanded (see state 1)
                   * No results text, or links to "what the law says about this" displays
 - [ ] Visually follows design:
      - [ ] Desktop version: https://zpl.io/VYkXxKE
      - [ ] Mobile version (see full set for all situations): https://zpl.io/bPvDeQk

### PR DESCRIPTION
**Pull request by YWHo**

### CODE REVIEW

File: packages/client/src/components/results/ResultAccordion/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    [DESKTOP]: '920px',
+    [TABLET]: '315px',
+    [MOBILE]: '315px'
+  })};
+`
+
+export default function ResultAccordion (props) {
+  const { format, topicObj = {} } = props
+  const { questions = [] } = topicObj
+  let isExpanded = true
```
Comment (chasm) [inline packages/client/src/components/results/ResultAccordion/index.js]: Why is this a `let`? Is it mutated somewhere?

File: packages/client/src/components/results/ResultAccordion/index.spec.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import * as React from 'react'
+import { create } from 'react-test-renderer'
+import ResultAccordion from './'
+import { DESKTOP, MOBILE, TABLET } from '../../../constants'
+
+describe('components:results:ResultAccordion', function () {
+  let topic
```
Comment (chasm) [inline packages/client/src/components/results/ResultAccordion/index.spec.js]: Not fond of this method, but we can leave it for now.

File: packages/client/src/components/results/ResultAccordion/index.spec.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  beforeEach(() => {
+    topic = {
```
Comment (chasm) [inline packages/client/src/components/results/ResultAccordion/index.spec.js]: If this never changes, why not just assign it to a `const` and use it, skipping the `beforeEach`?

File: packages/client/src/components/results/ResultTopicHeader/index.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
+import * as React from 'react'
+import styled from 'styled-components'
+import styledMap from 'styled-map'
+import { DESKTOP, MOBILE, TABLET } from '../../../constants'
+
+const { useState } = React
```
Comment (chasm) [inline packages/client/src/components/results/ResultTopicHeader/index.js]: Nice!

File: packages/client/src/components/results/ResultTopicHeader/index.js [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+    border-left: 4px solid transparent;
+    border-right: 4px solid transparent;
+  }
+`
+
+export default function ResultTopicHeader (props) {
+  const { format, name = '...', onClick, isExpanded } = props
+  const [expanded, setExpanded] = useState(isExpanded)
+
+  const showText = () => {
```
Comment (chasm) [inline packages/client/src/components/results/ResultTopicHeader/index.js]: These are recalculated every time the component is rendered, so why not `const`. What is the advantage of creating a function if you're going to call it immediately and it never changes?

File: packages/client/src/components/results/ResultTopicHeader/index.js [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+    return expanded ? 'Show Less' : 'Show More'
+  }
+
+  const showCaret = () => {
+    return expanded ? <CaretUp /> : <CaretDown />
+  }
+
+  const btnClicked = () => {
```
Comment (chasm) [inline packages/client/src/components/results/ResultTopicHeader/index.js]: This one is a callback, so it needs to be a function. The two above it do not.

File: packages/client/src/components/results/ResultAccordion/index.js (hunk 1 above)
Comment (YWHo) [reply packages/client/src/components/results/ResultAccordion/index.js]: Done

File: packages/client/src/components/results/ResultAccordion/index.spec.js (hunk 3 above)
Comment (YWHo) [reply packages/client/src/components/results/ResultAccordion/index.spec.js]: Done

File: packages/client/src/components/results/ResultTopicHeader/index.js (hunk 5 above)
Comment (YWHo) [reply packages/client/src/components/results/ResultTopicHeader/index.js]: Done

**Pull request merged**